src/installer.ts

```diff
@@ -54,7 +55,7 @@ export async function getGraalVM(
     graalvmShort = version.substr(0, version.indexOf('-dev-') + 4)
   }
   core.info(`Version of graalvm: ${version}, short: ${graalvmShort}`)
-  let toolPath = tc.find('GraalVM', getCacheVersionString(version), "amd64")
+  let toolPath = tc.find('GraalVM', getCacheVersionString(version), arch)
```

Comment:
We should also pass the `arch` variable when caching the directory below. It can be added as an optional parameter as the last parameter.

Reply:
It should be fixed now

Reply:
In my comment I was actually referring to line 92, where we are calling `tc.cacheDir()`. But indeed, we need to provide `arch` to `tc.findAllVersions()` as well.

Reply:
Right, fixed